config/customization.rb.template

```diff
@@ -0,0 +1,2 @@
+ADMIN_EMAIL = "betydb@gmail.com"
```

Comment:
I considered this but decided against it for a few reasons.  One is that the settings in the two files seem largely orthogonal, and I liked having a clearly-named place to put customization settings with a mind to possible expanding the available settings to encompass all of what might be required to address issue #368.  A less good reason was that I wasn't that familiar with `additional_environment.rb` and wanted to start somewhere fresh.  Third, I was thinking this may only be a temporary solution to an immediate problem, and if I'm going to eventually jettison this infrastructure in favor of something more comprehensive, it's nice to have those settings all in one place.  If you feel strongly about combining the files, I don't have a tremendous problem with doing so.